test/suites/dev/moonbase/test-staking/test-staking-locks11.ts

```diff
@@ -7,11 +7,11 @@ import {
   alith,
   generateKeyringPair,
 } from "@moonwall/util";
-import { chunk } from "../../../../helpers";
+import { chunk, getDelegatorStakingFreeze, getNumberOfDelegatorFreezes } from "../../../../helpers";
 
 describeSuite({
-  id: "D023476",
-  title: "Staking - Locks - bottom delegator removed",
+  id: "D013474",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.